### ISSUE TITLE
Improving the navigation in al_database

### ISSUE THREAD

**Issue by charlesduponpon** [cjekel/tindetheus#10]
> Use your personalized model to automatically like and dislike profiles. The profiles which you have automatically liked and disliked are stored in al_database.

In my case, all profiled liked and disliked automatically when doing `tindetheus like` are all dumped into the `al_database` directory. Would it be possible to have a separation done here? A like subdirectory and dislike subdirectory. Similar to the manner `database` works right now. We could quickly check if the model behaves as expected.

**Comment by cjekel** [comment]
This would be a very nice QOL improvement! 

It will break backwards compatibility... but it can be easy to create a script which mitigates this. 

In the meantime, you can use the validate function, and copy the ```al_database``` into a validation folder. This is wasteful in terms of space, but since the model is deterministic the results will be the same. The validate function will show 49 faces liked and 49 faces disliked, and then the rest will go into the csv file. 

```shell
mkdir validation
cp -r al_database validation/
tindetheus validate
```

I'll edit this with a script that you could run to copy the al_database into al_review folder for likes and dislikes.

**Comment by cjekel** [comment]
I created a script which will create a new ```al``` directory that will contain ```like``` and ```dislike``` folders. 

https://github.com/cjekel/tindetheus/blob/master/examples/make_al_like_dislike_folders.py

Not 100% sure about how to merge this in, as running this after each session would be wasteful if you've reviewed thousands of profiles. However, it is backwards compatible. Need to think about this more.

**Comment by cjekel** [comment]
I ended up including the functionality of that script as
```shell
tindetheus like_folder
```
which is available in the 0.4.3 release of tindetheus on pypi

**Comment by charlesduponpon** [comment]
Sweet thanks!
